packages/tldraw/src/lib/utils/tldr/buildFromV1Document.ts

```diff
@@ -564,18 +561,19 @@ export function buildFromV1Document(editor: Editor, document: LegacyTldrawDocume
 								})
 
 								if (change) {
-									if (change.props?.[handleId]) {
-										const terminal = change.props?.[handleId] as TLArrowShapeTerminal
-										if (terminal.type === 'binding') {
-											terminal.isExact = binding.distance === 0
-
-											if (terminal.boundShapeId !== targetId) {
-												console.warn('Hit the wrong shape!')
-												terminal.boundShapeId = targetId
-												terminal.normalizedAnchor = { x: 0.5, y: 0.5 }
-											}
-										}
-									}
+									// TODO(alex): #bindings can we delete this?
+									// if (change.props?.[handleId]) {
+									// 	const terminal = change.props?.[handleId] as TLArrowShapeTerminal
+									// 	if (terminal.type === 'binding') {
+									// 		terminal.isExact = binding.distance === 0
+
+									// 		if (terminal.boundShapeId !== targetId) {
+									// 			console.warn('Hit the wrong shape!')
+									// 			terminal.boundShapeId = targetId
+									// 			terminal.normalizedAnchor = { x: 0.5, y: 0.5 }
+									// 		}
+									// 	}
+									// }
```

Comment:
looks like there is some logic to add here for v1 backwards compatibility but it will look a bit different because we need to update the newly-created binding record rather than the 'change' returned from the shape util callback.